packages/theme/src/react/__specs__/with-theme.spec.js

```diff
@@ -1,54 +0,0 @@
-import { mount } from 'enzyme'
```

Comment:
Is this gone because we don't plan to support `withTheme`?  If so, shouldn't we keep it around until that moment we actually remove the `withTheme` src?

Reply:
yeah i can refactor to use testing-library for now and then we'll remove it when we remove `withTheme`